core/ui/SideBar/Open.tid

```diff
@@ -9,23 +9,29 @@ caption: {{$:/language/SideBar/Open/Caption}}
 <$action-listops $tiddler="$:/StoryList" $subfilter="+[insertbefore:currentTiddler<actionTiddler>]"/>
 \end
 
-<$list filter="[list[$:/StoryList]]" history="$:/HistoryList" storyview="pop">
-<div style="position: relative;">
-<$droppable actions=<<drop-actions>>>
-<div class="tc-droppable-placeholder">
-&nbsp;
+\define placeholder()
+\whitespace trim
+<div class="tc-droppable-placeholder" style="line-height:2em;height:2em;">
 </div>
```

Comment:
`</div>` can be on the same line. no content

Reply:
yes it can just be a `<div .../>`